### PR TITLE
Feature/adding artifactory publish ability

### PR DESCRIPTION
**Pull request by rcordeau**
This change is [Reviewable](https://reviewable.io/reviews/smarterapp/sbac-iat-rich-text-editor/2)

### CODE REVIEW

File: .npmrc [hunk 1]
```diff
@@ -0,0 +1,2 @@
+@sbac:registry=https://airdev.jfrog.io/airdev/api/npm/npm
```
Comment (rcordeau) [inline .npmrc:1]: This is just to point NPM at the right registry for publishing the Text Editor

File: .npmrc [hunk 2]
```diff
@@ -0,0 +1,2 @@
...
+//airdev.jfrog.io/airdev/api/npm/:_authToken=${NPM_AUTH_TOKEN}
```
Comment (rcordeau) [inline .npmrc:2]: NPM_AUTH_TOKEN is an environment variable and the CI server sets it on the build so it can publish.

Comment (rcordeau) [reply .npmrc:2]: I put the auth token on Confluence in case it is needed elsewhere.

**Review by alex-ponce — APPROVED** [review]
Changes look good

**Pull request merged**